snowfall/common2.py

```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/env python3
```

Comment:
common2.py is the same as common.py, except that it has some code supporting
the second pass model. To avoid conflicts with the master,  a new file is used.

The same goes for the following xxx2.py files, e.g., lm_rescore2.py, mmi2.py.